tests/xbot/common/properties/PropertyTest.java

```diff
@@ -120,4 +125,12 @@ public void testLoadingValueAfterCreation() {
         assertEquals(true,bool.get());
         assertEquals("teststring",str.get());
     }
+    
+    @After
+    public void cleanUp()
+    {
+    	// We need a way to obliterate the database locally so tests don't leak. Can't delete the files themselves,
+    	// because the database process still has a handle on some of them.
+    	assertEquals(true, ((PersonalComputerDatabaseStorage)propertyManager.permanentStore).obliterateStorage());
```

Comment:
I'm not sure if it's usual to put an assert in an After cleanup. It sounds like it would be an exception maybe if the storage wasn't successfully obliterated?